src/controller/CHIPDeviceController.cpp

```diff
@@ -1735,13 +1772,14 @@ void DeviceCommissioner::DisarmDone()
 
     // Signal completion - this will reset mDeviceBeingCommissioned.
     CommissioningStageComplete(CHIP_NO_ERROR);
-    SendCommissioningCompleteCallbacks(nodeId, mCommissioningCompletionStatus);
 
     // If we've disarmed the failsafe, it's because we're starting again, so kill the pase connection.
     if (commissionee != nullptr)
     {
         ReleaseCommissioneeDevice(commissionee);
     }
+
+    SendCommissioningCompleteCallbacks(nodeId, mCommissioningCompletionStatus);
```

Comment:
Is the ordering change here important?  If so, worth documenting why SendCommissioningCompleteCallbacks needs to come after ReleaseCommissioneeDevice.

Reply:
Added a comment